### ISSUE TITLE
Error on `debug_traceTransaction` `Unsupported JSON RPC version`

### ISSUE THREAD

**Issue by MysticRyuujin** [emeraldpay/dshackle#43]
Command:
```
curl -H "Content-Type: application/json" -d '{"id": 1, "method": "debug_traceTransaction", "params": ["0xd949bc0fe1a5d16f4522bc47933554dcc4ada0493ff71ee1973b2410257af9fe"]}' http://archive02.archivenode.io:8545
```

Output from node: 
[output.zip](https://github.com/emeraldpay/dshackle/files/4977913/output.zip)

Dshackle Error Log:
[ErrorLog.txt](https://github.com/emeraldpay/dshackle/files/4977915/ErrorLog.txt)

Interesting lines from the error log:
```
ERROR | HttpServerOperations | [id: 0x60d71fcc, L:/172.19.0.2:8545 - R:/172.19.0.3:48624] Error finishing response. Closing connection
    io.infinitape.etherjar.rpc.RpcException: RPC Error -32600: Unsupported JSON   RPC version
```

**Comment by MysticRyuujin** [comment]
Have you had a chance to look at this one yet?

**Comment by splix** [comment]
Hi, sorry, didn't have a chance yet. Will take a look soon

**Comment by splix** [comment]
The actual problem is that `jsonrpc` is not provided with the request. But Dshackle didn't respond with correct error message, fixed with 2abe302

**Comment by splix** [comment]
@MysticRyuujin please try v0.8.1 with the fix, just uploaded to Docker. Thank you
